hydra-cluster/config/protocol-parameters.json

```diff
@@ -368,19 +368,19 @@
     "maxValueSize": 5000,
     "minPoolCost": 0,
     "minUTxOValue": null,
-    "monetaryExpansion": 1.78650067e-3,
-    "poolPledgeInfluence": 0.1,
+    "monetaryExpansion": 0.1,
+    "poolPledgeInfluence": 0,
     "poolRetireMaxEpoch": 18,
     "protocolVersion": {
         "major": 7,
         "minor": 0
     },
-    "stakeAddressDeposit": 400000,
-    "stakePoolDeposit": 500000000,
-    "stakePoolTargetNum": 50,
+    "stakeAddressDeposit": 0,
+    "stakePoolDeposit": 0,
+    "stakePoolTargetNum": 100,
     "treasuryCut": 0.1,
-    "txFeeFixed": 0,
-    "txFeePerByte": 0,
-    "utxoCostPerWord": 34488,
-    "utxoCostPerByte": 4310
+    "txFeeFixed": 155381,
```

Comment:
This value must be 0

Reply:
interesting, I was thinking the default should be main-net costs.